app/main.py

```diff
@@ -1,7 +1,38 @@
+import os
+import requests
+from dotenv import load_dotenv
+
+load_dotenv()
+
 def get_weather() -> None:
```

Comment:
The function `get_weather` is defined to return `None`, but it actually returns a string. Update the function signature to reflect the correct return type.